food-truck-api/src/main/java/food/truck/api/recommendation/ScoringRecommendationStrategy.java

```diff
@@ -18,6 +21,9 @@
     private final User user;
     private final UserPreferences prefs;
 
+    // Since we're instantiating this with 'new', I can't @Autowire it
+    private final FoodTruckDictionary ftd = new FoodTruckDictionary();
+
```

Comment:
This means that we can't have anything injected inside FoodTruckDictionary, either. If we later need to autowire a service into FoodTruckDictionary, then we should create this as a bean and inject it in.

Reply:
I've modified this slightly to allow for some degree of field injection (as I decided was best to handle wiring in a WebClient).